src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/RoleResource.kt

```diff
@@ -36,12 +36,12 @@ class RoleResource(
   private val roleService: RoleService
 ) {
 
-  @PreAuthorize("hasRole('ROLE_MAINTAIN_ACCESS_ROLES_ADMIN')")
+  @PreAuthorize("hasAnyRole('ROLES_ADMIN','ROLE_MAINTAIN_ACCESS_ROLES_ADMIN')")
```

Comment:
this hasn't got a `ROLE_` prefix?

Reply:
i.e. should it be `ROLE_ROLES_ADMIN`?

Reply:
The integration test is still passing so might